project_template/forms.py

```diff
@@ -4,9 +4,16 @@
 from project_template import constants
 from project_template.datamodels.real_estate_type import RealEstateType
 from project_template.datamodels.attainment_type import AttainmentType
+from project_template.datamodels.mobile_goods_type import MobileGoodsType
+from project_template.datamodels.financial_institution import FinancialInstitution
+from project_template.datamodels.currency import Currency
+from project_template.datamodels.debt_type import DebtType
+from project_template.datamodels.IncomeProviderType import IncomeProviderType
+from project_template.datamodels.position import Position
 
-YEAR_CHOICES = ('2008', '2009', '2010', '2011', '2012', '2013', '2014', '2015', '2016', '2017', '2018', '2019')
 
+YEAR_CHOICES = ('2008', '2009', '2010', '2011', '2012', '2013', '2014', '2015', '2016', '2017', '2018', '2019', '2020', '2021', '2022', '2023', '2024', '2025')
+FIRST_2_TYPES = 2
```

Comment:
Not totally sure it is worth adding a name to this in order to do [0:FIRST_2_TYPES] instead of [0:2]

Reply:
I thought about it but it was already defined in Models, so I decided to copy it in Forms. The reason was like that: if it will be needed to be modified it's easier to change only the value submitted to the FIRST_2_TYPES variable.